index.html

```diff
@@ -1128,8 +1128,10 @@ <h3 class="heading settled" data-level="1.1" id="examples"><span class="secno">1
 <pre>&lt;script src="http://origin-b/widget.js">&lt;/script>
 </pre>
     <p>Before blocking this insecure request as mixed content, the user agent
-  performs an HSTS priming request to the root of the resource’s host, as
-  follows:</p>
+  checks whether the request <a data-link-type="dfn" href="#matches-the-hsts-cache">matches its HSTS
```

Comment:
> Nit: You can write this as `<a lt="matches the HSTS cache">matches its HSTS cache</a>`.

But I did! You're looking at the generated `index.html`. In `index.src.html`, I used the `lt` syntax, mimicking what I saw you had done in your draft:

https://github.com/mikewest/hsts-priming/pull/2/files#diff-117d6498d2aa8019cc0abf5eeb87a9faR133